File: Data Structures & Algorithms/sliding-window-maximum/submission-0.cpp
class Solution {
public:
    vector<int> maxSlidingWindow(vector<int>& nums, int k) {
        int n = nums.size();
        vector<int>result;
        int left = 0 , right = left + k ;

        while(right <= n ){
            int maxE = INT_MIN;
            for(int i = left ; i < right ; i++){
                maxE = max(maxE,nums[i]);
            }
            result.push_back(maxE);
            left++;
            right++;
        }
        return result;
    }
};

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.go
func findDuplicate(nums []int) int {
    l , r:= 0 , 1
    for l < len(nums)- 1 {
      if nums[l]== nums[r]{
        return nums[l]
      }
      r++

      if r == len(nums){
        l++
        r = l+1
      }
    }
    return -1
}
